aiida/restapi/resources.py

```diff
@@ -204,6 +204,122 @@ def get(self, id=None, page=None):  # pylint: disable=redefined-builtin,invalid-
         return self.utils.build_response(status=200, headers=headers, data=data)
 
 
+class QueryBuilder(BaseResource):
+    """
+    "Abstract" QueryBuilder REST API resource.
+
+    It is abstract in the sense that it supports POST requests taking in JSON QueryBuilder query help
+    objects and returning the QueryBuilder result accordingly.
+    """
+    from aiida.restapi.translator.nodes.node import NodeTranslator
+
+    _translator_class = NodeTranslator
+    _parse_pk_uuid = 'uuid'  # Parse a uuid pattern in the URL path (not a pk)
```

Comment:
I guess the comment of this field does not apply here

Reply:
I'm starting to think I don't need this variable altogether. Will try to write some tests now and figure it out.